pkg/action/diff.go

```diff
@@ -99,6 +120,52 @@ func processSrc(ctx context.Context, c malcontent.Config, src, dest map[string]*
 	}
 }
 
+func processDest(ctx context.Context, c malcontent.Config, from, to map[string]*malcontent.FileReport, d *malcontent.DiffReport) {
```

Comment:
I moved these functions around to make them easier to reference. They're coupled enough that jumping back and forth was a little painful.